spring-session-data-redis/src/main/java/org/springframework/session/data/redis/RedisIndexedSessionRepository.java

```diff
@@ -288,7 +288,7 @@
 
 	private byte[] expiredKeyPrefixBytes;
 
-	private final RedisOperations<Object, Object> sessionRedisOperations;
+	private final RedisOperations<String, Object> sessionRedisOperations;
```

Comment:
Forgot to point out this after opening the PR - this is technically a minor breaking change but it was needed in order to align the base configuration support - `RedisSessionRepository` uses `RedisOperations<String, Object>` from the start, and that's more exact since our ids are strings anyway. However, I don't think it's a big deal considering we're also changing the session repository which `@EnableRedisHttpSession` provides.

Reply:
Thanks I noticed this. I think this is an acceptable change especially given this is a major release anyway.